plugins/webapi/tools/message/send_message.go

```diff
@@ -17,10 +17,6 @@ const maxIssuedAwaitTime = 5 * time.Second
 
 // SendMessage is the handler for tools/message endpoint.
 func SendMessage(c echo.Context) error {
-	if !deps.Tangle.Synced() {
```

Comment:
The node needs to be bootstrapped to be able to issue a block. Is this checked in some other place?

Reply:
Fixed